components/content/inspira/ui/text-generate-effect/TextGenerateEffect.vue

```diff
@@ -1,20 +1,15 @@
 <template>
-  <div class="font-bold">
-    <div class="mt-4">
-      <div class="dark:text-white text-black text-2xl leading-snug tracking-wide">
-        <div ref="scope">
-          <span
-            v-for="(word, idx) in wordsArray"
-            :key="word + idx"
-            class="dark:text-white text-black inline-block"
-            :style="spanStyle"
-          >
-            {{ word }}&nbsp;
-          </span>
-        </div>
-      </div>
-    </div>
-  </div>
+  <div class="leading-snug tracking-wide">
```

Comment:
Please add a prop `class` to the component, so that when user applies class to component, then are managed by `cn()` method.

You can refer other components for the implementation

Reply:
@rahul-vashishtha done